src/stores/configStore.js

```diff
@@ -217,6 +218,11 @@ export class ConfigStore {
       } else if (key.includes('valid')) {
         // Convert the boolean config values to booleans
         this[key] = Boolean(value);
+      } else if (key === 'customSequences'){
+        // Store customSequences as an array of strings
+        value = encodeURIComponent(value);
```

Comment:
Make sure this doesn't literally re-encode already encoded characters

If this is necessary, just write a comment about why